core/modules/system/config/system.core.json

```
@@ -57,7 +57,9 @@
     "rss_description": "",
     "rss_limit":  10,
     "rss_viewmode": "fulltext",
+    "site_403_enabled": 0,
     "site_403": "",
+    "site_404_enabled": 0,
     "site_404": "",
```

I still don't like these 😅 ...whether we show the system-provided 403/404 pages or the custom ones, these will be enabled. I would ideally like these to be:

```suggestion
    "custom_403_page": 0,
    "custom_403_path": "",
    "custom_404_page": 0,
    "custom_404_path": "",
```

I realize that we'd need an update hook in order to move these, but I think that it'd be worth it from a DX point. I'll let others chime in to see if I'm the only one that feels this way.